apps/console/src/features/applications/components/forms/inbound-saml-form.tsx

```diff
@@ -483,54 +483,63 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
-                                    ".placeholder")
-                            }
-                            validationErrorMsg={
-                                t("console:develop.features.applications.forms.inboundSAML.fields.assertionURLs" +
-                                    ".validations.invalid")
-                            }
-                            validation={ (value: string) => {
+                        <Grid.Row columns={ 1 }>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                <URLInput
+                                    urlState={ assertionConsumerUrls }
+                                    setURLState={ setAssertionConsumerUrls }
+                                    labelName={
+                                        t("console:develop.features.applications.forms.inboundSAML.fields" +
+                                            ".assertionURLs.label")
+                                    }
+                                    value={ initialValues?.assertionConsumerUrls.toString() }
+                                    placeholder={
+                                        t("console:develop.features.applications.forms.inboundSAML.fields" +
+                                            ".assertionURLs" +
+                                            ".placeholder")
+                                    }
+                                    validationErrorMsg={
+                                        t("console:develop.features.applications.forms.inboundSAML.fields" +
+                                            ".assertionURLs" +
+                                            ".validations.invalid")
+                                    }
+                                    validation={ (value: string) => {
 
-                                let label: ReactElement = null;
+                                        let label: ReactElement = null;
 
-                                if (!URLUtils.isHttpsOrHttpUrl(value)) {
-                                    label = (
-                                        <Label basic color="orange" className="mt-2">
-                                            { t("console:common.validations.unrecognizedURL.description") }
-                                        </Label>
-                                    );
-                                }
+                                        if (!URLUtils.isHttpsOrHttpUrl(value)) {
+                                            label = (
+                                                <Label basic color="orange" className="mt-2">
+                                                    { t("console:common.validations.unrecognizedURL.description") }
+                                                </Label>
+                                            );
+                                        }
```

Comment:
why the indentation here? Could we remove that?

Reply:
The indentation was added since we added 
```
<Grid.Row columns={ 1 }>
      <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
```
to wrap the URL Input component.
https://github.com/wso2/identity-apps/blob/a6c6c50b921c3a122131c22efecc05da73b2e6fb/apps/console/src/features/applications/components/forms/inbound-saml-form.tsx#L486-L487